org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/angular/AngularLanguageServer.java

```diff
@@ -50,7 +50,7 @@ public AngularLanguageServer() {
 
 	@Override protected ProcessBuilder createProcessBuilder() {
 		ProcessBuilder builder = super.createProcessBuilder();
-		builder.environment().put("NG_DEBUG", Boolean.toString(true));
```

Comment:
We should keep the NG_DEBUG set for cases when the preference to log and Angular LS is true.

Reply:
I revert this modification here and I will update #332